### PR TITLE
SwiftPM support. Dropped carthage to simplify conversion.

### PR DESCRIPTION
**Pull request by melle**

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -81,15 +81,59 @@ You can test the broadcast and the handler for receiving messages by running the
...
+    .package(url: "https://github.com/teufelaudio/UDPBroadcastConnection.git", .branch("master"))
+  ],
+  targets: [
+    .target(name: "MyApp", dependencies: ["UDPBroadcastConnectionDynamic"])
+  ]
+)
 ```
 
-Then run `carthage update`.
+Then you can either building on the terminal or use Xcode 11 or higher that now supports SPM natively.
```
Comment (zweigraf) [inline ReadMe.md]: typo: `build`

File: Sources/UDPBroadcastExample/UDPBroadcastExample.xcodeproj/project.xcworkspace/contents.xcworkspacedata [hunk 2]
```diff
@@ -2,6 +2,6 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "self:UDPBroadcast.xcodeproj">
+      location = "self:UDPBroadcastExample.xcodeproj">
```
Comment (zweigraf) [inline Sources/UDPBroadcastExample/UDPBroadcastExample.xcodeproj/project.xcworkspace/contents.xcworkspacedata:5]: It looks like the UDPBroadcastExample is in two workspaces? Should one be the UDPBroadcast project?

**Review by zweigraf — APPROVED** [review]

**Review by luizmb — APPROVED** [review]

**Comment by luizmb** [comment]
This is usually my .gitignore for SPMs (it's Github default one, removing Xcodeprojs and workspaces, you may want to keep xcodeproj and workspace because of the example app):
```
# Xcode
#
# gitignore contributors: remember to update Global/Xcode.gitignore, Objective-C.gitignore & Swift.gitignore

## User settings
xcuserdata/

## compatibility with Xcode 8 and earlier (ignoring not required starting Xcode 9)
*.xcscmblueprint
*.xccheckout

## compatibility with Xcode 3 and earlier (ignoring not required starting Xcode 4)
build/
DerivedData/
*.moved-aside
*.pbxuser
!default.pbxuser
*.mode1v3
!default.mode1v3
*.mode2v3
!default.mode2v3
*.perspectivev3
!default.perspectivev3

## Obj-C/Swift specific
*.hmap

## App packaging
*.ipa
*.dSYM.zip
*.dSYM

## Playgrounds
timeline.xctimeline
playground.xcworkspace

# Swift Package Manager
#
# Add this line if you want to avoid checking in source code from Swift Package Manager dependencies.
Packages/
Package.pins
Package.resolved
*.xcodeproj
#
# Xcode automatically generates this directory with a .xcworkspacedata file and xcuserdata
# hence it is not needed unless you have added a package configuration file to your project
.swiftpm

.build/

# CocoaPods
#
# We recommend against adding the Pods directory to your .gitignore. However
# you should judge for yourself, the pros and cons are mentioned at:
# https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
#
# Pods/
#
# Add this line if you want to avoid checking in source code from the Xcode workspace
# *.xcworkspace

# Carthage
#
# Add this line if you want to avoid checking in source code from Carthage dependencies.
# Carthage/Checkouts

Carthage/Build/

# Accio dependency management
Dependencies/
.accio/

# fastlane
#
# It is recommended to not store the screenshots in the git repo.
# Instead, use fastlane to re-generate the screenshots whenever they are needed.
# For more information about the recommended setup visit:
# https://docs.fastlane.tools/best-practices/source-control/#source-control

fastlane/report.xml
fastlane/Preview.html
fastlane/screenshots/**/*.png
fastlane/test_output

# Code Injection
#
# After new code Injection tools there's a generated folder /iOSInjectionProject
# https://github.com/johnno1962/injectionforxcode

iOSInjectionProject/
```

**Comment by zweigraf** [comment]
For gitignore, SPM also creates an appropriate gitignore file when you run

`swift package init/create` in the directory.

**Pull request merged**